integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
Without the log message, the information about the failure appears only in the testng test results information, not in the test output.  I find it extremely convenient when diagnosing problems to have information about the problem in both places.  But since the log messages has\ve caused repeated code review comments, I will cease doing that.